File: app/src/main/java/rssdemo/andras/hu/rssdemo/RssDemoApplication.java
package rssdemo.andras.hu.rssdemo;

import android.app.Application;

import rssdemo.andras.hu.rssdemo.di.Injector;

public class RssDemoApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        Injector.init(this);
    }
}

File: app/src/main/java/rssdemo/andras/hu/rssdemo/RssDemoApplication.kt
package rssdemo.andras.hu.rssdemo

import android.app.Application

import rssdemo.andras.hu.rssdemo.di.Injector

class RssDemoApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        Injector.init(this)
    }
}
